spec/support/matchers/accessibility.rb

```diff
@@ -310,13 +327,11 @@ def fieldset_legend_name(element)
 end
 
 def expect_page_to_have_no_accessibility_violations(page, validate_markup: true)
-  expect(page).to be_axe_clean.according_to(
-    :section508, :"best-practice",
-    :wcag21aa
-  ).
+  expect(page).to be_axe_clean.according_to(:wcag2aa, :"best-practice").
```

Comment:
One of these includes `:section508`?

Reply:
Yes, see extended commit description of eb3522b3640daa3de7c9a6a2c79fa033fba64eff for details.

Reply:
Bike-shedding here but does bumping the standard level to wcag22aa result in a bunch of warnings/failures? In some very quick local tests I'm not seeing anything additional come up.

Reply:
I changed it from `:wcag21aa` to `:wcag2aa` hoping that would just use whatever the latest 2.x version is (including 2.2 if supported by Axe), so yes I would ideally like to target 2.2

I should double-check that `:wcag2aa` actually means "latest 2.x" and not "2.0 specifically" though.

Reply:
> I should double-check that `:wcag2aa` actually means "latest 2.x" and not "2.0 specifically" though.

Turns out `:wcag2aa` _does_ refer to 2.0 specifically ([ref](https://github.com/dequelabs/axe-core/blob/master/doc/API.md#axe-core-tags)), so I updated to `:wcag22aa` in 462ae07. I'll see if any other issues crop up from the upgrade.